chromium_src/chrome/install_static/brave_product_install_details_unittest.cc

```diff
@@ -204,6 +204,7 @@ TEST_P(MakeProductDetailsTest, DefaultChannel) {
 TEST_P(MakeProductDetailsTest, UpdateAp) {
   // This test is only valid for brands that integrate with Google Update.
 #if !defined(OFFICIAL_BUILD)
+  if ((true))
     return;
 #endif
```

Comment:
I think we should use `MAYBE_` macro for these tests instead of the early return.

Reply:
Fixed.

Reply:
`UpdateAp` test is not fixed.

Reply:
Fixed.